sql/src/main/java/org/apache/druid/sql/calcite/schema/SegmentMetadataHolder.java

```diff
@@ -72,9 +74,14 @@ public String getSegmentId()
     return segmentId;
   }
 
-  public long getNumReplicas()
+  public ImmutableMap<String, ImmutableSet<String>> getsegmentServerMap()
```

Comment:
nit: incorrect camel casing. Additionally, I think the naming of this should be symmetrical with `withReplicas`, so either rename this `getReplicas`, or rename the other `withSegmentServerMap` maybe.

Reply:
renamed to `getReplicas`